### PR TITLE
Accessibility exercise (2nd Review)

### PR DESCRIPTION
**Pull request by utsargo**
This is the branch for the accessibility exercise of @utsargo and @codehass .
We have checked all the required criteria. The report is given below.

**Checklist**
**- Page titles**: OK
**- Image text alternatives**: OK. All images have their text alternatives.
**- Text headings**: We found an unnecessary H2 in contact section. Changed that into a paragraph. [Link](https://github.com/portfolio-roy/portfolio-roy.github.io/blob/accessibility-exercise/index.html#L244-L247)
**- Color contrast**: OK
**- Resize**: OK
**- Interaction**: OK 
**- Moving content**: OK
**- Multimedia**: Not applicable
**- The basic structure of the page**: OK

**A mistake came to our knowledge after the first review. We made that right. Here is the [link](https://github.com/portfolio-roy/portfolio-roy.github.io/blob/accessibility-exercise/index.html#L177).**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,269 +1,392 @@
...
-                    </fieldset>
-                    <input type="email" name="email" placeholder="Email Adress" required>
-                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
-                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
-                </form>
+      </section>
+      <section class="about-me" id="about">
+        <div class="section-content container">
+          <div class="about-text">
+            <p class="section-heading txt-center">About Me</p>
```
Comment (omarsalem7) [inline index.html]: - [x] Kindly make this title `h2` to make these work for everyone, the headings need to be marked up. That way people can navigate to the headings — including people who cannot use a mouse and use only the keyboard, and people who use a screen reader.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @utsargo, @codehass 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (utsargo) [reply index.html]: Ok, my partner mistakenly edited the wrong tag. I'm on it.

**Review by omarsalem7 — APPROVED** [review]
Hi team😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**